Makefile

```diff
@@ -108,21 +106,24 @@ INSTALL_CORE_FILES = \
     stbt-screenshot \
     stbt-tv
 
-all: $(INSTALL_CORE_FILES)
+all: stbt.sh defaults.conf stbt/__init__.py $(INSTALL_CORE_FILES) \
```

Comment:
This rule at line 51:

    extra/fedora/stb-tester.spec stbt.sh stbt/__init__.py: %: %.in ...